server/core/src/https/apidocs/response_schema.rs

```diff
@@ -61,27 +61,27 @@ impl IntoResponses for ApiResponseWithout200 {
 }
 
 #[derive(Debug, Clone, ToSchema)]
-// TODO: this should be `webauthn_rs_proto::auth::PublicKeyCredential``, but ... I don't know how to make it possible in utoipa
+// this should be `webauthn_rs_proto::auth::PublicKeyCredential``, but ... I don't know how to make it possible in utoipa
```

Comment:
I don't think we should even worry, because webauthn is not an API that people should be mucking with in scripts. We need to be more careful about what apis *are* and *are not* for things to use.